components/usage/pkg/apiv1/usage_test.go

```diff
@@ -563,6 +563,10 @@ func TestUsageService_ReconcileUsageWithLedger(t *testing.T) {
 	to := time.Date(2022, 05, 1, 1, 00, 00, 00, time.UTC)
 	attributionID := db.NewTeamAttributionID(uuid.New().String())
 
+	t.Cleanup(func() {
+		require.NoError(t, dbconn.Where("attributionId = ?", attributionID).Delete(&db.Usage{}).Error)
```

Comment:
We expect the controller to create Usage records with this Attribution ID, so we clean them up here in the hook